nodejs/week1/src/backend/routes/meals.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const filepath = require('path').resolve(__dirname, '..');
+
+
+function getReview(id) {
+    let revData = fs.readFileSync(`${filepath}/data/reviews.json`);
+    let reviews = JSON.parse(revData);
+    const reviewArray = [];
+
+    reviews.forEach(element => {
+        if(element.mealId == id) {
+            reviewArray.push(element);
+        }
+    });
+    return reviewArray;
+}
+
+function mealsRouter() {
+    let mealData = fs.readFileSync(`${filepath}/data/meals.json`);
+    let meals = JSON.parse(mealData);
+    
+    meals.forEach(element => {

```

Comment:
use `.map` instead of `.forEach` - my humble preference but I think most would agree